broker/service/schema_service.py

```diff
@@ -0,0 +1,22 @@
+import ast
+import json
+import jsonref
+
+
+class SchemaService(object):
+    def __init__(self):
+        self.json_loader = jsonref.JsonLoader()
+
+    def dereference_schema(self, json_schema):
```

Comment:
Might be a good idea to annotate method declarations with types, like `def dereference_schema(self, json_schema: dict) -> JsonRef` or something.